juniper_official/interface/check-ifl-stats.rule

```diff
@@ -0,0 +1,295 @@
+                term is-ifl-stats-normal {
+                    when {
+                        less-than "$out-util" "$low-threshold";
+                    }				
+                    then {
+                        status {
+                            color green;
+                            message "$interface-name.$sub-interface-index output traffic:$out-mbps mbps, util:$out-util %, out-bw: $out-bw mbps is normal";
+                        }
+                    }
+                }				
+                /*
+                 * Sets color to yellow and sends out an anomaly notification
+                 * when the subinterface output traffic ($out-util)
+                 * is above low threshold (low-threshold)
+                 */				
+                term is-ifl-stats-median {
+                    when {
+                        greater-than-or-equal-to "$out-util" "$low-threshold";
+                        less-than "$out-util" "$high-threshold";
+                    }
+                    then {
+                        status {
+                            color yellow;
+                            message "$interface-name.$sub-interface-index output traffic:$out-mbps mbps, util:$out-util %, out-bw: $out-bw mbps is in medium range";
+                        }
+                    }
+                }
+                /*
+                 * Sets color to red and sends out an anomaly notification when
+                 * the interface output traffic ($out-util) is
+                 * above high threshold (high-threshold)
+                 */ 				
+                term is-ifl-stats-abnormal {
+                    when {
+                        greater-than-or-equal-to "$out-util" "$high-threshold";
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "$interface-name.$sub-interface-index output traffic:$out-mbps mbps, util:$out-util %, out-bw: $out-bw mbps is above high threshold";
+                        }
+                    }
+                }				
+            }
+            variable ifd-name {
+                value ".*";
+                description "Interfaces to be monitored, regular expression, eg 'ge-.*'";
+                type string;
+            }
+            variable ifl-no {
+                value ".*";
+                description "Sub interfaces index to be monitored, regular expression, eg '0-10'";
+                type string;
+            }
+            variable traffic-high-threshold {
+                value 80;
+                description "Subinterface traffic high threshold value in percentage";
+                type int;
+            }
+            variable traffic-low-threshold {
+                value 50;
+                description "Subinterface traffic high threshold value in percentage";
+                type int;
+            }
+            variable out-bw {
+                value 512;
```

Comment:
yes, it will be always different in most of the services. output bandwidth will be always lower than input at customer side. only in datacenter's it will be vice versa.